### DISCUSSION TITLE
Is it possible to use mapsui in blazor

### DISCUSSION THREAD

**Discussion by peder1896** [Mapsui/Mapsui#1709] [Q&A]
Is it possible to use mapsui in blazor now that skiasharp is supported
Thanks for anay tips.

**Comment by inforithmics** [comment]
Theoretically yes at the moment no.
But for that I think a Mapsui.UI.Blazor Target needs to be made.
https://www.nuget.org/packages/SkiaSharp.Views.Blazor

Mapsui itself already runs in WASM with the help of Uno.wasm and uno.Winui.wasm so the task necessary would be to create a Mapsui.UI.Blazer target. The changes in mapsui are already done (Don't use Task.Result or Task.Wait blocks wasm).
Don't how how feasible or easy this would be.

**Reply by p2pbsh** [reply]
This would make a superb addition to the project. Currently available Blazor map components aren't particularly impressive, having become rather smitten with mapsui, would love to be able to use it in a Blazor wasm app.

**Comment by Sanasaa** [comment]
> Is it possible to use mapsui in blazor now that skiasharp is supported Thanks for anay tips.

I have no clue what this websites about. But someone aka an evil person down lo passed mapsui on my phone and it's a spyware app yes. Can someone plea see e help me to remove it I'm desperate.. I don't want to live like this any mm ore

**Comment by inforithmics** [comment]
Pull request for Mapsui.Blazor

https://github.com/Mapsui/Mapsui/pull/1747

**Reply by p2pbsh** [reply]
Absolute legend

**Reply by abhaymin** [reply]
Hi,

Is there an ETA by when a stable editions of this component will be out or a roadmap?

**Reply by pauldendulk** [reply]
@abhaymin Since this an open source project and the time we have for this project depends on other circumstances as well it is hard to give estimates. Also in the past my estimates on Mapsuis roadmap have been wrong by a large margin. Yet, to give you some idea, I think it will be al least six weeks before the first release candidate. See the roadmap here: https://mapsui.com/documentation/roadmap.html.
